backend/models/distance.py

```diff
@@ -132,7 +133,9 @@ def closest(self, origin, point_list):
             enumerate(distance_responses),
             key=operator.itemgetter(1)
         )
-        # TODO: Determine the origin of these magic numbers to convert seconds to miles.
+        if self.mode == 'time':
+            return min_distance / 60.0, point_list[min_idx]
+        # TODO: Determine the origin of these magic numbers to convert seconds to meters.
         return min_distance * 104607.0 / 3600, point_list[min_idx]
```

Comment:
So all other values for `self.mode` default to distance? Could be clearer to have `if self.mode == 'distance'`...

Reply:
Hopefully split into separate functions soon.